File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/LockupEntrypoint/Item_.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Item_ {

    @SerializedName("count")
    @Expose
    private Integer count;
    @SerializedName("average")
    @Expose
    private Integer average;
    @SerializedName("via")
    @Expose
    private Via_ via;
    @SerializedName("supplier")
    @Expose
    private Supplier_ supplier;
    @SerializedName("attribution")
    @Expose
    private String attribution;

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public Integer getAverage() {
        return average;
    }

    public void setAverage(Integer average) {
        this.average = average;
    }

    public Via_ getVia() {
        return via;
    }

    public void setVia(Via_ via) {
        this.via = via;
    }

    public Supplier_ getSupplier() {
        return supplier;
    }

    public void setSupplier(Supplier_ supplier) {
        this.supplier = supplier;
    }

    public String getAttribution() {
        return attribution;
    }

    public void setAttribution(String attribution) {
        this.attribution = attribution;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/Item_.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import com.google.gson.annotations.Expose

class Item_ {
    @SerializedName("count")
    @Expose
    private var count: Int? = null

    @SerializedName("average")
    @Expose
    private var average: Int? = null

    @SerializedName("via")
    @Expose
    private var via: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Via_? = null

    @SerializedName("supplier")
    @Expose
    private var supplier: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Supplier_? = null

    @SerializedName("attribution")
    @Expose
    private var attribution: String? = null
    fun getCount(): Int? {
        return count
    }

    fun setCount(count: Int?) {
        this.count = count
    }

    fun getAverage(): Int? {
        return average
    }

    fun setAverage(average: Int?) {
        this.average = average
    }

    fun getVia(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Via_? {
        return via
    }

    fun setVia(via: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Via_?) {
        this.via = via
    }

    fun getSupplier(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Supplier_? {
        return supplier
    }

    fun setSupplier(supplier: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Supplier_?) {
        this.supplier = supplier
    }

    fun getAttribution(): String? {
        return attribution
    }

    fun setAttribution(attribution: String?) {
        this.attribution = attribution
    }
}